packages/admin-panel-server/src/routes/ImportDashboardVisualisationRoute.ts

```diff
@@ -70,12 +97,17 @@ export class ImportDashboardVisualisationRoute extends Route<ImportDashboardVisu
   };
 
   private createVisualisation = async (visualisation: Record<string, unknown>) => {
-    const { id } = await this.meditrakConnection.createResource(
-      'dashboardVisualisations',
-      {},
-      visualisation,
-    );
-    return id;
+    await this.meditrakConnection.createResource('dashboardVisualisations', {}, visualisation);
```

Comment:
It's a bit inconvenient that our meditrak-server mutation endpoints do not return the affected resources...